src/patternfly/components/Table/table-th.hbs

```diff
@@ -20,22 +20,24 @@
   {{#if table-th--attribute}}
     {{{table-th--attribute}}}
   {{/if}}>
-  {{#if table-th--sortable}}
-    {{#> button button--modifier="pf-m-plain"}}
-      {{> @partial-block}}
-      <span class="pf-c-table__sort-indicator">
-        {{#if table-th--selected}}
-          {{#if table-th--asc}}
-            <i class="fas fa-long-arrow-alt-up"></i>
+  {{#> table-cell-content}}
```

Comment:
Do you need this in the `th` elements? The content element is only needed for the mobile view, and `thead` is hidden on mobile.